app/(gcforms)/[locale]/(form administration)/form-builder/[id]/settings/components/branding/Branding.tsx

```diff
@@ -68,6 +66,7 @@ export const Branding = ({ hasBrandingRequestForm }: { hasBrandingRequestForm: b
     const operationResult = await updateTemplate({
       id,
       formConfig,
+      allowPublishedUpdate: true,
```

Comment:
Thoughts on making a more granular action for updateBranding  where we get the current val from the db and take in a param for just the branding --- merge that into the exiting template.

Thinking something like that would offer more protection as to only updating part of a published template